eng/pipelines/branch-cleanup.yml

```diff
@@ -2,16 +2,22 @@ parameters:
 - name: Repos
   type: object
   default:
-    - azure-sdk-for-android
-    - azure-sdk-for-c
-    - azure-sdk-for-cpp
-    - azure-sdk-for-go
-    - azure-sdk-for-ios
-    - azure-sdk-for-java
-    - azure-sdk-for-js
-    - azure-sdk-for-net
-    - azure-sdk-for-python
-
+    - Azure/azure-sdk-for-android
```

Comment:
This is what is causing the eng/common sync pipeline to break. 'Azure' is already appended as $PROwner for the sparse checkout and other places. So you need to not add `Azure` here.

Reply:
I make the changes to [eng/pipelines/eng-common-sync.yml](https://github.com/Azure/azure-sdk-tools/pull/3061/files#diff-b5d7243f40e5fa06023749f464cfd8b8b32485140592e31036ceb7a2b6a2f7fd)

Does it help?

Reply:
Also change the interface Delete-RemoteBranch to take the repo id